### ISSUE TITLE
Pelican 3.7 will change the way Jinja Extensions are loaded

### ISSUE THREAD

**Issue by minchinweb** [MinchinWeb/minchin.pelican.jinja_filters#1]
See https://github.com/getpelican/pelican/pull/1999

**Comment by minchinweb** [comment]
Implementation example here -> https://github.com/getpelican/pelican-plugins/pull/825

**Comment by justinmayer** [comment]
I don't think this issue is relevant any longer, but if I am mistaken, please feel free to comment here so we can re-open it.
